react/src/components/ManageMapProjectPanel/MapTabContent.tsx

```diff
@@ -3,7 +3,7 @@ import { Project, ProjectRequest } from '@hazmapper/types';
 import { SectionMessage } from '@tacc/core-components';
 import { EditFilled, CheckOutlined, DeleteOutlined } from '@ant-design/icons';
 import { Button, Flex, List, Input, Modal } from 'antd';
-import { useAppConfiguration } from '@hazmapper/hooks';
+// import { useAppConfiguration } from '@hazmapper/hooks';
```

Comment:
same comment as https://github.com/TACC-Cloud/hazmapper/pull/365/files#r2198833599 for this line